portfolio/src/main/webapp/about-me/about-me.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link href="https://fonts.googleapis.com/css2?family=Cookie&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap" rel="stylesheet">
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="../default.css">
+    <link rel="stylesheet" href="../buttons.css">
+    <link rel="stylesheet" href="about-me.css">
+  </head>
+  <body>
+    <div class="photo-frames">
+      <img src="images/selfie.jpg" alt="Avatar"/>
```

Comment:
nitpick: the tailing dash (/) is not necessary for html5 anymore. You might want to remove it, as mentioned in https://google.github.io/styleguide/htmlcssguide.html#Document_Type.

Reply:
According to this style guide I should also remove closing tag for <p> and <head> tag. Although I'm afraid my code will look like spaghetti if I remove all optional end tags and catching bugs will be harder.
https://stackoverflow.com/questions/3008593/html-include-or-exclude-optional-closing-tags
What do you think? These optimisations have its reason in improving network speed, but in this case (code is really small) it wouldn't change much I think.

Reply:
Also <img/> tag is removed, marking as done.